index.js

```diff
@@ -56,5 +58,11 @@ class BootSimulator {
   }
 }
 
+let myArgs = require('optimist').argv;
```

Comment:
can we have this at the top level

Reply:
@saikrishna321 Is it ok if i move the if statement also with this on top?